RHEL6_7/packages/RemovedPackages/check

```diff
@@ -32,6 +43,12 @@ echo \
 the upgrade for some of your packages. We are not aware of any compatible
 replacement for these packages.
 
+Use the '--cleanup-post' option of redhat-upgrade-tool to remove them
+automatically during upgrade. You can also run manually the $NOAUTO_POST_DIR/$NOAUTO_SCRIPT_NAME script
+after the upgrade to remove only the original native packages with broken
```

Comment:
- *run manually the XY script* **->** *run the XY script manually*
- *only old native packages with broken dependencies* **->** *only those RHEL 6 packages that have broken dependencies*

Reply:
fixed